canonical-web-and-design/ubuntu.com
# Revert change to our-philosophy redirect

## Done
Reverted erroneous change to our-philosophy redirect and applied it to the licence URL. 

## QA
- Check the code changes are expected
- Visit the URLs and ensure they do not error
